end-to-end-tests/features/violations/message_too_long.feature

```diff
@@ -0,0 +1,29 @@
+Feature: When the commit title exceeds the configured maximum length, it is picked up as a linting violation.
+
+
+  Scenario: Commit message that passes default length limit but fails with custom limit
```

Comment:
Change the scenarios so there is one test that starts with a too long message. You run lifting it fails then you add the arg to make it larger than 50 and it passes. The other it starts with a too small and it passes lifting, then you run it with the arg making it smaller than 50 and lifting fails.